### PR TITLE
Local env

### PR DESCRIPTION
**Pull request by frankwiles**
Bunch of changes to README and setup instructions

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -98,6 +105,30 @@ The ksvotes.org site makes Kansas online voting registration easy.
...
+
+  The encryption key is kind of particular, it needs to be 32 bytes long and URl-safe base64 encoded.  Use this command to generate one for you using the cryptography library: 
+
+  ```
+  $(venv) make crypt-key
+  ```
+
+### Demo uuid
+
+  We need `DEMO_UUID` set to a UUID, use this to generate one for your quickly:
```
Comment (karpet) [inline README.md]: typo?

File: README.md [hunk 2]
```diff
@@ -123,6 +154,8 @@ Navigate to [localhost:5000](http://localhost:5000)
 
 ## Tests
 
+**NOTE**: You need a local instance [NVRIS](https://github.com/BlueprintKansas/NVRIS) setup for the tests to pass.
```
Comment (karpet) [inline README.md]: or set your `NVRIS_URL` to the stage instance that Travis CI uses.

File: config.py [hunk 3]
```diff
@@ -2,7 +2,8 @@
 
 import os
 from dotenv import load_dotenv
-load_dotenv()
+load_dotenv(dotenv_path=".env", verbose=True)
```
Comment (karpet) [inline config.py]: weird that the default must be explicitly set but fine with me.

Comment (frankwiles) [reply config.py]: Yeah I agree, not sure what is up with that.

**Pull request merged**